NewForce-Cohort-6/tabloidfullstack-cranberry-sauce
# Lr5 details

As a reader, I would like to see the content of a Post so I can read it.

Given a user is viewing a list of Posts
When they select a post to read
Then they should be directed to a Post Detail page that shows the Post Details.

Post Details include:

Title
Header image (if exists)
Content
Author's Display Name


Testing

1. Commit or stash whatever you're currently working on
2. Git fetch --all
3. Git checkout LR5Details
4. git pull origin LR5Details
5. [create database here](https://github.com/NewForce-Cohort-6/tabloidfullstack-cranberry-sauce/blob/main/SQL/01_Tabloid_Create_DB.sql)
6. [seed data here](https://github.com/NewForce-Cohort-6/tabloidfullstack-cranberry-sauce/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Start database from inside Visual Studio by pressing the green play button
8. Run npm start from inside Tabloid client folder.
9. Login using an email that you know is there and works for you (example: [foo@bar.com](mailto:foo@bar.com) from seed data)
10. Once logged-in, use the navbar to navigate to post management ("/post")
11. Click on a post to view its details. 
12. Confirm that the post displays ... 
Title
Header image (if exists)
Content
Author's Display Name
13. Choose post management again to navigate back to the post list page.
